### ISSUE TITLE
feat: hotreload when template, configuration file is changed

### ISSUE THREAD

**Issue by lukasmrtvy** [powerman/dockerize#150]
would be great to implement support for hotreload ( if it is explicitily allowed via ENV variable for example ) application when template, configuration file is changed

reference: https://github.com/weaveworks/Watch 

thanks

**Comment by powerman** [comment]
Not really. It's just a helper to run apps in a docker, solving usual issues related to containers, like:

- needs to wait until required services (like MySQL) starts in another containers before running the app
- needs to copy environment variable values into app's config before starting the app
- needs to stream app's log file to stderr

I.e. dockerize takes the app which wasn't designed for use in containers and makes it easier to use it in containers. In 99% cases dockerize won't even continue to runs after starting the app.
